src/test/run-make-fulldeps/coverage-reports/expected_show_coverage.partial_eq.txt

```diff
@@ -2,15 +2,15 @@
     2|       |// structure of this test.
     3|       |
     4|      2|#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord)]
-                                                    ^1       ^1
+                       ^0            ^0      ^0 ^0  ^1       ^0 ^0^0
```

Comment:
Cool that we get coverage for these, but what's going on with the second counters under PartialEq and Ord?

Reply:
I don't really know how they get expanded.

I'll add a FIXME comment and issue. I don't think this is high priority, but it might be nice to clean up in the future.

Reply:
Issue #79626

Reply:
Added issue and referenced it in a comment in the test.